javascript/rest-nextjs/package.json

```diff
@@ -1,29 +1,27 @@
 {
-  "name": "hello-next",
-  "version": "1.0.0",
-  "description": "",
-  "keywords": [],
-  "license": "MIT",
-  "author": "",
-  "main": "index.js",
+  "name": "rest-nextjs",
+  "version": "0.1.0",
+  "private": true,
   "scripts": {
-    "dev": "next",
+    "dev": "next dev",
     "build": "next build",
-    "start": "next start"
+    "start": "next start",
+    "lint": "next lint"
   },
   "dependencies": {
-    "@prisma/client": "5.1.1",
+    "@prisma/client": "^5.1.1",
     "next": "13.4.16",
-    "next-superjson": "0.0.4",
     "react": "18.2.0",
     "react-dom": "18.2.0",
-    "react-markdown": "8.0.7",
-    "superjson": "1.13.1"
+    "react-markdown": "^8.0.7",
+    "supports-color": "^8.1.1"
   },
   "devDependencies": {
-    "prisma": "5.1.1"
+    "@types/node": "20.5.0",
```

Comment:
super nitpick and ignore if you want but generally `js` repos won't have types deps